unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -64,7 +70,6 @@ export class DgraphDataSource extends RESTDataSource {
       query (func: eq(type, "drug")) @cascade {
         code
         description
-        type
```

Comment:
Wha? I don't remember doing this. Guess can fix it in the merge...

Reply:
🙀